dbms/src/Storages/MergeTree/BoolMask.cpp

```diff
@@ -0,0 +1,5 @@
+#include "BoolMask.h"
+
+
+const BoolMask BoolMask::consider_only_can_be_true(false, true);
```

Comment:
Ok. BTW, C++ also support `inline data member`:
```
static inline constexpr BoolMask ...
```

Reply:
No, it does not as BoolMask is an incomplete struct by the moment of inline definition. 

Illustration: https://wandbox.org/permlink/81IEwqYd5JXvkh4Q